src/main/java/school/lemon/changerequest/java/introduction/hw2/Task8.java

```diff
@@ -1,8 +1,20 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
 public class Task8 {
-
     public static int[] evenOdd(int[] arr) {
-        return null;
+        if (arr == null) {
+            return null;
+        }
+        int lastEven = 0;
+        for (int i = 0; i < arr.length; i++) {
```

Comment:
Good solution but it doesn't safe order. 
You don't need to do it in one loop and do not need to modify current array.